databinding/build.gradle

```diff
@@ -26,5 +26,5 @@ android {
 }
 
 dependencies {
-    compile 'com.github.acrimi.AndroidUtils:core:core_0.0.14'
-}
\ No newline at end of file
+    compile 'com.github.ebbnormal.AndroidUtils:core:core_0.0.26'
```

Comment:
Revert dependency

Reply:
so how will this work, i just point to the old dependencies, and then you will slowly create tags of each one starting with core (Which has the changes), and then update the tags with the new tag you make?

Reply:
yes, dependencies are updated when new releases are created